nixos/tests/dsl.nix

```diff
@@ -220,8 +251,13 @@ in
       systemd.tmpfiles.settings."10-expressions" = {
         "/run/dag-test/nix-dag-test.json"."L+".argument = "${configJson}";
         "/run/dag-test/scope.nix"."L+".argument = "${expressions}";
+        "/run/dag-test/requestScope.nix"."L+".argument = "${requestExpressions}";
```

Comment:
H'm why another file with expressions? All the data is already in `scope.nix`, which I ended up not using anyway

Reply:
Oh, I see, you're using this as the entrypoint for evanix... I guess it's OK. Maybe we rename `scope.nix` into `all-packages.nix` and `requestScope.nix` into `targets.nix`, or something like that

Reply:
i didn't have to add that for the test, but it's easier to debug